### ISSUE TITLE
Don’t understand how true_score and false_score are represented in logits

### ISSUE THREAD

**Issue by rubaha96** [shmsw25/FActScore#17]
factscore/factscorer.py lines 219-220, in _get_score:

true_score = logits[5852]
false_score = logits[7700]

Is there any intuition about these particular logit indices and how logits are formed?

**Comment by shmsw25** [comment]
Hi @rubaha96, these are vocab ids for `True` and `False` (in LLAMA tokenizer). So these scores represent how likely the model will complete the sequence with `True` or `False`.
